client/src/pages/dashboard/views/Staff.svelte

```diff
@@ -68,21 +69,26 @@
     {#await staffReady}
         <p>Loading staff page...</p>
     {:then}
+        <header>
         <h1>Staffs of {officeName}</h1>
-        {#each $staffList.filter(s => s.permission !== 0) as { id, name, email, permission, picture } (id)}
-            <PersonRowLocal
-                {id}
-                {email}
-                {name}
-                {permission}
-                {picture}
-                office={currentOffice}
-                iconSize={IconSize.Large} 
-                on:overflowClick={openContextMenu.bind(null, id, currentOffice, email, permission)} 
-            />
-        {:else}
-            <p>No staff members exist in "{officeName}".</p>
-        {/each}
+        <!-- TODO: Put addStaff button here. -->
+        </header>
+        <Container ty={ContainerType.Enumeration}>
+            {#each $staffList.filter(s => s.permission !== 0) as { id, name, email, permission, picture } (id)}

```

Comment:
I'm beginning to think that this looks cleaner.
```suggestion
            {@const staff = $staffList.filter(s => s.permission !== 0)}
            {#each staff as { id, name, email, permission, picture } (id)}
```